### PR TITLE
Updates in Prescient server

### PR DESCRIPTION
**Pull request by kcho**
Now Prescient network is running two separate lochness instances, one for mock participant and another for real participants. Configuration file now accepts a location of the CSV file with the list of mock case IDs. It also accepts a location of a CSV file with the list of IDs to ignore. Email alert functions are also updated in accordance with this change.

### CODE REVIEW

File: lochness/utils/path_checker.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def update_interviews_transcript_check(df: pd.DataFrame) -> pd.DataFrame:
     '''Check logics in rows for Interviews transcripts'''
     transcript_int_index = df[
             (df.modality=='Interviews') &
-            (df.subject.isin(['For_review', 'Approved']))].index
+            (df.subject.isin(['For_review', 'For review', 'Approved']))].index
```
Comment (tashrifbillah) [inline lochness/utils/path_checker.py:92]: Did you knowingly duplicate `For review`?

File: lochness/rpms/__init__.py [hunk 2]
```diff
@@ -329,6 +362,7 @@ def sync(Lochness, subject, dry=False):
                                BIDS=Lochness['BIDS'])
         proc_dst = Path(proc_folder) / f"{subject_id}_{measure}.csv"
 
+        get_run_sheets_for_datatypes(target_df_loc)
```
Comment (tashrifbillah) [inline lochness/rpms/__init__.py:365]: Were we not extracting RPMS run sheets so far?

File: scripts/sync.py [hunk 3]
```diff
@@ -102,6 +102,8 @@ def main():
                              'server side')
     parser.add_argument('-ds', '--daily_summary', action='store_true',
                         help='Enable daily summary email function')
+    parser.add_argument('-cs', '--check_source', action='store_true',
+                        help='Enable check source email function')
```
Comment (tashrifbillah) [inline scripts/sync.py:106]: The help message is unclear. I am unable to propose an alternative because I am not sure what the underlying function does.

**Comment by tashrifbillah** [comment]
> It also accepts a location of a CSV file with the list of IDs to ignore. 

So will you transfer only the 5 mock cases moving forward?
https://redcap.partners.org/redcap/redcap_v12.0.19/DataEntry/record_status_dashboard.php?pid=34678

**Comment by tashrifbillah** [comment]
This PR seems to be lingering for a while. Why do we not merging it? My line-by-line review is probably no longer required.

**Comment by tashrifbillah** [comment]
This PR seems to be lingering for a while. Why do we not merge it? My line-by-line review is probably no longer required.

File: lochness/utils/path_checker.py (hunk 1 above)
Comment (kcho) [reply lochness/utils/path_checker.py:92]: Thanks for catching this.

Comment (kcho) [reply lochness/utils/path_checker.py]: [5035314](https://github.com/AMP-SCZ/lochness/pull/112/commits/50353147a1bc9bd1eb26d21ab16d0f2e3bb2c8d0)

File: lochness/rpms/__init__.py (hunk 2 above)
Comment (kcho) [reply lochness/rpms/__init__.py:385]: We have been extract the run sheets from RPMS for a long time, but the commits and push were delayed on my end. Thanks.

File: lochness/utils/source_check.py [hunk 4]
```diff
@@ -367,7 +429,7 @@ def get_subject_list_from_metadata(Lochness: 'lochness') -> List[str]:
 
 
 def get_all_rpms_subjects_with_consent(Lochness) -> pd.DataFrame:
-    '''Return df with subject column of all subjects IDs with consentID'''
+    '''Get df with the 'subject' column of subjects IDs from metadata file'''
```
[the remark this answers is not in the document]
Comment (kcho) [reply lochness/utils/source_check.py:432]: No, it's within the triple quotation mark, which allow you use the single quotation without the backslash.

File: scripts/sync.py (hunk 3 above)
Comment (kcho) [reply scripts/sync.py:106]: It goes through existing files on the data sources like `box`, `xnat`, `mediaflux` etc. and checks if the file conforms to the AMP-SCZ SOP file naming. If any file deviates from the SOP pattern, these out of SOP files would be curated in a table and sent to the networks by email.

**Comment by kcho** [comment]
@tashrifbillah Apologies, my review on this PR was significantly delayed. I'll review this branch once more when we agree to merge https://github.com/AMP-SCZ/lochness/pull/113. Thanks.

**Pull request merged**